restapi/src/main/java/io/stargate/web/models/UdtAdd.java

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright The Stargate Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.stargate.web.models;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.google.common.base.MoreObjects;
+import io.stargate.web.models.udt.UdtType;
+import io.swagger.annotations.ApiModelProperty;
+import java.util.List;
+import javax.validation.constraints.NotNull;
+
+@JsonInclude(JsonInclude.Include.NON_NULL)
+public class UdtAdd {
+  private boolean ifNotExists;
+  private @NotNull List<UdtType> fields;
+
+  @ApiModelProperty(
+      value =
+          "Determines whether to create a new udt if an udt with the same name exists. Attempting to create an existing udt returns an error unless this option is true.")
```

Comment:
Might get confusing in the swagger if we talk about `udt` when the path is `/types`
```suggestion
          "Determines whether to create a new type if a type with the same name exists. Attempting to create an existing type returns an error unless this option is true.")
```